### DISCUSSION TITLE
Move a calc line up and down

### DISCUSSION THREAD

**Discussion by gotfredsen** [kazad/instacalc#74] [Q&A] [answered]
Is there a way to move the calc line up and down, like a destination in google maps?

**Comment by kazad** [comment, marked as answer]
Hi! Yes, you can drag and drop a line by pressing on the line number, for example:

https://github.com/kazad/instacalc/assets/115572/364e0981-7a4f-428b-b71e-6d432f2bab3c

Hope that helps!

**Comment by gotfredsen** [comment]
Thanks, it works perfect on safari on my MacBook, but couldn't get it to work on my iPad

**Comment by kazad** [comment]
Ah, I believe it is only enabled for devices without touchscreens. I'll have to see if there is a library that supports it well (without accidental misdrags for tapping).
